src/Form/Select.js

```diff
@@ -121,6 +121,7 @@ class Select extends Component {
         <SelectContainer value={value} size={size}>
           <SelectInput
             {...props}
+            size={size}
```

Comment:
Another FF quirk: select options are grayed until you actually select one. 

<img width="791" alt="Screen Shot 2019-08-19 at 10 35 18 AM" src="https://user-images.githubusercontent.com/6404663/63286549-25a05880-c26d-11e9-8bbb-19d49cc0fd0f.png">